dashboard/lib/tasks/seed.rake

```diff
@@ -154,6 +154,63 @@ namespace :seed do
     seed_regional_partners_school_districts(true)
   end
 
+  task create_pd_dashboard_data: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    # Destroy existing facilitator and organizers
+    User.where(email: ['pd_test_facilitator@code.org', 'pd_test_organizer@code.org']).destroy_all
+
+    # Make a facilitator
+    facilitator = create(:facilitator, email: 'pd_test_facilitator@code.org')
+    Pd::CourseFacilitator.create(facilitator_id: facilitator.id, course: Pd::Workshop::COURSE_CSP)
+
+    # Make a workshop organizer
+    organizer = create(:workshop_organizer, email: 'pd_test_organizer@code.org')
+
+    # Create an ended workshop
+    workshop = create(:pd_workshop, course: Pd::Workshop::COURSE_CSP, num_sessions: 1, enrolled_and_attending_users: 5, organizer: organizer)
+    workshop.facilitators << facilitator
```

Comment:
Note you can inline the facilitator in the `create` via `facilitators: [facilitator]`. Same below.